CoreUtilities/src/au/gov/asd/tac/constellation/utilities/color/ConstellationColor.java

```diff
@@ -330,6 +330,65 @@ public static ConstellationColor fromHtmlColor(final String color) {
         return ConstellationColor.getColorValue(red / 255f, green / 255f, blue / 255f, 1f);
     }
 
+    /**
+     * Get a contrasting color based on YIQ values. Will return BLACK if the
+     * passed in color is null.
+     *
+     * @param color HTML color to contrast with
+     * @return the ConstellationColor which contrasts with color.
+     */
+    public static ConstellationColor getContrastHtmlColor(final String color) {
```

Comment:
These methods are great! Can you please document these additional methods in `CHANGELOG.md`?

Reply:
I already had I believe. Do you want me to further expand on this?
`Added utility methods to `ConstellationColor` which assist with getting inverse colors.`

Reply:
> 
> 
> I already had I believe. Do you want me to further expand on this?
> `Added utility methods to `ConstellationColor` which assist with getting inverse colors.`

No that's perfect, I missed it.